tests/userUpdateAvatarRequestEndpoint.test.js

```diff
@@ -1,31 +1,44 @@
-const startServer = require("../server");
+const startServer = require("../src/server");
 const request = require("supertest");
 const should = require("should");
-const { UnauthorizedError } = require("../helpers/usersHelpers");
-const userModel = require("../users/userModel");
-const {
-  Types: { ObjectId },
-} = require("mongoose");
+const { UnauthorizedError } = require("../src/helpers/usersHelpers");
+const userModel = require("../src/users/userModel");
+
+const jwt = require("jsonwebtoken");
+require("dotenv").config();
 
 describe("Acceptance tests for users endpoint case", () => {
-  let server;
+  describe("PATCH /api/v1/users/avatars", () => {
+    let server;
+    const userId = "5f1f31dd172af82b8cf91182";
+    const token = jwt.sign({ id: userId }, process.env.JWT_SECRET);
 
-  before(async () => {
-    process.env.SILENT = false;
-    server = await startServer();
-  });
+    before(async () => {
+      process.env.SILENT = false;
+      server = await startServer({
+        connection: process.env.MONGODB_CONNECTION_URL_FOR_TESTS,
+      });
 
-  after(() => {
-    server.close();
-    process.env.SILENT = true;
-  });
+      await userModel.create({
+        _id: userId,
+        subscription: "free",
+        email: "alex.code@gmail.com",
+        password:
+          "$2a$06$iqoFleFNyRZG/P/7Xh0i2O5hFOZXADsl0G9MvgVGJjW52V.wu4zQK",
```

Comment:
you should create password like:

const password = 'mysecretpass123';

then you should encrypt password

then use encrypted password in await userModel.create({